proto/substrait/algebra.proto

```diff
@@ -112,9 +115,12 @@ message ReadRel {
       // the length in byte to read from this item
       uint64 length = 8;
 
-      enum FileFormat {
-        FILE_FORMAT_UNSPECIFIED = 0;
-        FILE_FORMAT_PARQUET = 1;
+      // The format of the files
+      oneof file_format {
+        google.protobuf.Empty parquet = 9;
```

Comment:
> I suggest we create empty message types for each of these. That would make more sense to people, I think. E.g. message ParquetFormat {}

Done.  Please check this to make sure I understood your point.

> Let's update format to be marked as deprecated rather than deleting (for now). We can batch up removal of deprecations when we want to bump the format version.

I was hoping that "breaking changes == minor" means that we no longer need to worry about batching breaking changes.  Let me know if something further is needed here.